articles/connections/social/google.md

```diff
@@ -14,6 +14,8 @@ seo_alias: google
 
 To connect your Auth0 client to Google and Google Apps, you will need to generate a *Client ID* and *Client Secret* in a Google project, copy these keys into your Auth0 settings, and enable the Connection.
 
+If you are looking to manage authentication in your native Android app [see the Android quickstart](/quickstart/native/android).
+
```

Comment:
@angelaLazar I think a Next Steps paragraph looks good (the user set up the connection, tested it, and now wants to use it, he needs to implement something so the Next Steps make sense). It doesn't sound right to add something out of context because one user made one wrong click.

Let's discuss this in our call today.